src/QuiqChatClient.js

```diff
@@ -260,7 +257,8 @@ class QuiqChatClient {
     const newEvents = differenceBy(events, this.events, 'id');
 
     if (newMessages.length && this.callbacks.onNewMessages && sendNewMessageCallback) {
-      this.callbacks.onNewMessages(newMessages);
+      // $FlowIssue - flow doesn't believe thie below callback is defined at this point
```

Comment:
You can do:
```javascript
const sortedMessages = sortByTimestamp(newMessages);
if (this.callbacks.onNewMessages) {
  this.callbacks.onNewMessages(sortedMessages);
}
```
I think the `sortByTimestamp` method call makes flow not trust the null check anymore